restic/genericCommand.go

```diff
@@ -115,7 +116,7 @@ func (g *genericCommand) exec(args []string, options commandOptions) {
 	}
 }
 
-func (g *genericCommand) collectOutput(output io.ReadCloser, print bool) ([]string, error) {
+func (g *genericCommand) collectOutput(output io.ReadCloser, print bool, out chan string) ([]string, error) {
 	collectedOutput := make([]string, 0)
 	scanner := bufio.NewScanner(output)
```

Comment:
A `bufio.Reader` might have been the better choice here. See https://golang.org/pkg/bufio/#Reader.ReadLine.

Reply:
I decided to create a separate ticket to tackle this issue. As the whole output management stuff needs some radical overthinking now that most commands can return json outputs.

But this should be addressed before a new release is made as there could be some pretty heavy memory leaks currently.